01_getting_started/hello_world.py

```diff
@@ -49,13 +46,13 @@ def f(i):
 #
 # Now let's see three different ways we can call that function:
 #
-# 1. As a regular `local` call on your computer, with `f.local`
+# 1. As a regular `local` call on your computer, with `f.local()`
 #
-# 2. As a `remote` call that runs in the cloud, with `f.remote`
+# 2. As a `remote` call that runs in the cloud, with `f.remote()`
 #
-# 3. By `map`ping many copies of `f` in the cloud over many inputs, with `f.map`
+# 3. By _mapping_ many copies of `f` in the cloud over a sequence of inputs, with `f.map()`
```

Comment:
[Why waste time say lot word when few do trick](https://media1.tenor.com/m/IsYdPRq7bjcAAAAC/why-waste-time-when-few-word-do-trick.gif)?

I don't think any target reader would be take the non-technical term "many" to mean we need `.map` to call a function with multiple inputs.

Reply:
Trying to explain what a map is (a sequence of inputs), not everyone knows what map is